static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.ts

```diff
@@ -28,10 +31,16 @@ const setSlotAdRefresh = (
 				// Determine whether ad should refresh
 				// This value will then be checked when the timer has elapsed and
 				// we want to know whether to refresh
-				advert.shouldRefresh = shouldRefresh(
+				const shouldSlotRefresh = shouldRefresh(
 					advert,
 					nonRefreshableLineItemIds,
 				);
+
+				advert.shouldRefresh = shouldSlotRefresh;
+
+				if (shouldSlotRefresh) {
+					setAdSlotMinHeight(advert);
+				}
```

Comment:
Would the the coupling between `memoizedFetchNonRefreshableLineItemIds` and `setAdSlotMinHeight` mean we only set the min-height once we have the non-refreshable-line-items?

This might work out okay in practice but I was wondering if we could decouple by setting min-height regardless and if it doesn't refresh then thats okay?

Reply:
I'll give that a try. What I tried before, was to not exclude outstream ads specifically and to use `shouldRefresh` as an indicator to whether or not to add the min-height. That didn't work due to passbacks, so I have to exclude oustream ads. I'll test removing the `shouldRefresh` condition and see if I run in to any problems

Reply:
It works without the `shouldRefresh` condition 🙌 .

The ignore outstream rule is imperative though.